### ISSUE TITLE
Section custom height

### ISSUE THREAD

**Issue by danmo** [subtirelumihail/react-fullpage#62]
Hi,
I have a question where a section's height is about double the viewport's height. This messes up the entire slide experience. 

Is there any way to set custom section height instead of the pre-calculated one?

**Comment by daydream05** [comment]
hey are you also having a problem where you're having an extra empty section?

**Comment by danmo** [comment]
@daydream05 Yes, something similar. I have a lot of extra space after the sections.

**Comment by qpixHassaballah** [comment]
i have the same issue whenever i scroll down to the next section , the bottom of the previous section appears which messes up the view drastically

**Comment by qpixHassaballah** [comment]
Anyone has any clue on how to decrease the previous section size ?

**Comment by qpixHassaballah** [comment]
Check your classes inside, if you are using react-bootstrap some predefined styles interfere with how full page .js work .
Mine was Jumbotron , once i removed it the bug disappeared

**Comment by ShailendraAccubits** [comment]
> hey are you also having a problem where you're having an extra empty section?

hey, were you able to solve it? i am having same problem
